### ISSUE TITLE
VGGCurve loading VGGBase object: curve_model.load_state_dict(checkpoint['state_dict'])

### ISSUE THREAD

**Issue by BelhalK** [timgaripov/dnn-mode-connectivity#7]
Good morning,

I'm having an error while compiling
```
curve_model.load_state_dict(checkpoint['state_dict'])
```
returning that there are `Missing key(s) in state_dict`, in particular  `"net.layer_blocks.0.0.weight_0"`.
It makes sense since `curve_model` is a `VGGCurve` object and my checkpoint model is a classical `VGG` net trained using my own training script.

How can `curve_model.load_state_dict(checkpoint['state_dict'])` work since `checkpoint['state_dict']` and `curve_model` have different keys?

Thank you very much for this.

**Comment by izmailovpavel** [comment]
Hey @BelhalK. You shouldn't be able to load a standard vgg checkpoint. If you want to use the code to visualize the loss for a standard checkpoint, you need to modify the code.

**Comment by BelhalK** [comment]
Good morning,
I have trained a VGG19 using my custom optimization method + a baseline method.
I now want to use your plane.py script to plot the both methods iterates on a 2D loss landscape.

The VGG19 was trained using ad-hoc torch scripts.

**Comment by izmailovpavel** [comment]
Right, so as I said, the script wouldn't work as is. You need to look through the code and modify it according to what you want to do.
